icons/beaker.svg

```diff
@@ -0,0 +1,15 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M4.5 3h15"/>
```

Comment:
Can you put a white space before the closing the tag?
```suggestion
  <path d="M4.5 3h15" />
```

Reply:
Should i do that to all of them?

Reply:
Yes please

Reply:
I realized that the icons from #118 are missing this spacing as well. Could I sneak the changes for these icons into this PR?

Reply:
Hmm, I think it's better to create a new PR for that one that's easier to track the changes